client/src/components/FloatingActionButton/FloatingActionButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
```

Comment:
you have removed react-router-dom package from the package.json in this PR, will this throw an error?

Reply:
@nimish-gupta while resolving conflict the package was at two locations in package.json so removed one of them